src/stylesheets/reactions.scss

```diff
@@ -8,14 +8,16 @@
   overflow-x: hidden;
   overflow-y: visible;
   > .reaction-button:last-child {
-    border-right: none!important;
+    border-right: 1.5px !important;
   }
 }
 
 .reaction-list > .reaction-button {
   font-weight: normal;
   padding: $spacer-2 $spacer-3;
   border-radius: 0!important;
+  border-right-width: 0px !important;
+  border-left-width: 1px !important;
```

Comment:
This is a little tricky because primer button groups show/hide left and right borders based on focus/active/hover:
https://github.com/primer/css/blob/master/src/buttons/button-group.scss#L35-L45

Forcing border visibility via `!important` will cause some problems with keyboard navigation:
![image](https://user-images.githubusercontent.com/1306747/53707715-f3955a00-3de4-11e9-886c-70b1f92622b4.png)

Reply:
@jdanyow Hmm, not sure this is the best fix anymore.  I opened issue #130 with a couple other ideas and might give this another go later this week.  Feel free to close this PR if needed.